great-idea/index.html

```diff
@@ -9,15 +6,83 @@
+  <!--[if lt  IE 9]>
+    <script  src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
 
-  <body>
+<body>
+  <div class="container"> 
+    <header>
+      <nav>
+        <a href="services.html">Services</a>
+        <a href="#">Product</a>
+        <a href="#">Vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+        <a class="logo-link" href="#"><img src="img/logo.png" alt="Great Idea! Company logo."></a>
+      </nav>
+    </header>
+
+    <div class="banner-container">
+      <section class="slogan-container">
+        <h1>Innovation</br>On</br>Demand</h1></br>
+        <a href="#">Get Started</a>
+      </section>
+      <img class="hero-image" src="img/header-img.png" alt="Image of a code snippet.">
+    </div>
 
-    <!-- Copy and paste your HTML from the first UI project here -->
+    <div class="content-container">
+      <div class="content-row-1">
+        <section>
+          <h2>Features</h2>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+                Suspendisse
+                eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </section><!--
+    --><section>
+            <h2>About</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+                Suspendisse
+                eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </section>
+      </div>
+
+      <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
       
-    
-  </body>
+      <div class="content-row-2">
+        <section>
+          <h2>Services</h2>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+          Suspendisse
+          eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </section><!--
+     --><section>
+          <h2>Product</h2>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+            Suspendisse
+            eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </section><!--
+     --><section>
+          <h2>Vision</h2>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+            Suspendisse
+            eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </section>
+      </div>
+    </div>
+
+    <footer>
+      <address>
+        <h2>Contact</h2>
+        <p>123 Way 456 Street</p>
+        <p>Somewhere, USA</p></br>
```

Comment:
`p` tags work, but `address` is probably more semantic. Furthermore, you can enclose the whole address within `address` and still force separate lines through `br`:
```
<address>
  123 Way 456 Street<br>
  Somewhere, USA
</address>
```

Reply:
Thank you, great tip. I'll note this for the future.